src/pages/WalletManagement/WalletWordsPage.tsx

```diff
@@ -16,9 +16,10 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { walletGenerate } from '@alephium/sdk'
```

Comment:
Not really in favor of not using our sdk function and duplicating its code in here.

Reply:
Ah yes; this is an artifact of an earlier revision